src/axom/slic/interface/slic.cpp

```diff
@@ -16,7 +16,7 @@ namespace slic
 // Initialize static variables for controlling runtime behavior of asserts and
 // error macros.
 //------------------------------------------------------------------------------
-AXOM_API bool debug::checksAreErrors = false;
+bool debug::checksAreErrors = false;
```

Comment:
Why is this removed?

Reply:
I did that.  

msvc 2019 had about 100 link warnings on this line 
(LNK4286 -- https://docs.microsoft.com/en-us/cpp/error-messages/tool-errors/linker-tools-warning-lnk4286?view=msvc-160)

I think it is related to `dllimport`/`dllexport` for a function that is being used in the same object library.
We should confirm that this will not cause an issue with any of our Windows users, and if so, maybe add a `#pragma` to silence the warning.

Reply:
all right